src/layouts/EditPage/MarkdownEditPage.tsx

```diff
@@ -119,11 +120,14 @@ export const MarkdownEditPage = ({ togglePreview }: MarkdownPageProps) => {
     updateHtmlWithMediaData()
   }, [mediaData, editorValue, updateHtmlWithMediaData])
 
+  const { isAnyDrawerOpen } = useEditorDrawerContext()
+
   return (
     <EditPageLayout
       variant="markdown"
       getEditorContent={() => editorValue}
       setEditorContent={(content: string) => setEditorValue(content)}
+      shouldDisableSave={isAnyDrawerOpen}
```

Comment:
this is the old editor right? if yes can default to yes leh, since this drawer only applies to cards + details (new rte)

Reply:
this is the markdown editor, prior to using tiptap. this is **feature flagged** but yea, as noted in the writeup, should be safe to enable but left as-is to maintain compat

Reply:
can also